news/2024/2024-12-14-osu-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,583 @@
+---
+layout: post
+title: osu! World Cup 2024 Concludes
+date: 2024-12-14 22:30:00 +0000
+---
+
+They've all come so far, but there can only be one Champion.
```

Comment:
capitalized champion looks weird there imo
```suggestion
They've all come so far, but there can only be one champion.
```

Reply:
i wrote it that way exactly because i wanted the capitalization on that word

Reply:
i got bullied into changing it https://github.com/ppy/osu-wiki/pull/12559/commits/c18604628bcfe913b20bdc203532ae1fcc3991c6